cmd/tsbs_generate_queries/databases/timescaledb/devops.go

```diff
@@ -56,6 +57,14 @@ func (d *Devops) getHostWhereString(nhosts int) string {
 	return d.getHostWhereWithHostnames(hostnames)
 }
 
+func (d *Devops) getTimeBucket(seconds int) string {
+	if d.UseTimeBucket {
+		return fmt.Sprintf("time_bucket('%d seconds', time)", seconds)
+	}
+
+	return fmt.Sprintf("to_timestamp(((extract(epoch from time)::int)/%d)*%d)", seconds, seconds)
```

Comment:
I feel like using `date_trunc` is probably the more natural thing to do here, though admittedly that complicates the interface and won't work, e.g., if we decide to queries on 5-min or non-singular intervals. What do you think?

Reply:
Think it depends on what to achieve. As long as the benchmark only needs minute/hour precision, I am happy to use `date_trunc`. Otherwise I prefer the timestamp.

Reply:
Okay, we'll leave it like this, since it gives us more flexibility in time groupings